common/Makefile

```diff
@@ -44,7 +44,7 @@ GEN_LD_DIR := $(SOC_SOFTWARE_DIR)/include/generated
 LDSCRIPT   := $(LD_DIR)/linker.ld
 LDSCRIPTS  := $(LDSCRIPT) $(GEN_LD_DIR)/output_format.ld $(GEN_LD_DIR)/regions.ld
 
-SRC_DIR    := $(BUILD_DIR)/src
+SRC_DIR    := src
```

Comment:
We are in `$(BUILD_DIR)`, right?   What's the motivation for this change?

Reply:
Before this change, to identify a file by path, the linker script would need to include the absolute path name.